src/connectors/CustomNetworkConnector.ts

```diff
@@ -38,7 +38,7 @@ class CustomMiniRpcProvider implements AsyncSendable {
   }
 
   public readonly sendAsync = (
-    request: { jsonrpc: '2.0'; id: number | string | null; method: string; params?: unknown[] | object },
+    request: { jsonrpc: '2.0'; id: number | string | null; method: string; params?: unknown[] },
```

Comment:
I usually use `Record<string, any>` instead of `object`. Thoughts?

Reply:
Its little different as per my knowledge. `object` type accepts `[] and ()=>{}` but `Record` doesn't. Record only accepts an  `{}` . Depends on what we want. Here the type is too much vague, need to make it specific. 
Here since params can be an object as well we can do `Record<"string", unknown>`

Since its vague even without that `object` type it doesn't complain since again its been typecasted to `any` and that was bad.

Most cases `Record<'string', any>` is the way to go. 👍🏼

Reply:
Will do a refactor in another PR. But thanks for pointing. I think as of now its ok. Since object is no more a type in new version of typescript,  it complain thats why I removed. :)

Reply:
Perfect. Thank you!